src/js/intl/strings.js

```diff
@@ -413,6 +442,8 @@ exports.strings = {
     'zh_CN': '你所指定的参数不兼容或者不准确',
     'zh_TW': '您指定了不相容或錯誤的選項',
     'es_AR': 'Las opciones que especificaste son incompatibles o incorrectas',
+    'es_ES': 'Las opciones que especificaste son incompatibles o incorrectas.',
+    'es_AR': 'No es necesario hacer add a los archivos en esta demo',
```

Comment:
whoops found a few dupe keys, this is preventing pushing the site. ill try to fix this